site-src/concepts/api-overview.md

```diff
@@ -1,3 +1,31 @@
 # API Overview
 
-TODO
\ No newline at end of file
+## Bakcground
+The Gateway API Inference Extension project is an extension of the Kubernetes Gateway API for serving Generative AI models on Kubernetes. Gateway API Inference Extension facilitates standardization of APIs for Kubernetes cluster operators and developers running generative AI inference, while allowing flexibility for underlying gateway implementations (such as Envoy Proxy) to iterate on mechanisms for optimized serving of models. 
+
+<img src="/images/inference-overview.png" alt="Overview of API integration" class="center" width="700" />
+
+
+## Key Features 
+Gateway API Inference Extension, along with a reference implementation in Envoy Proxy, provides the following key features: 
+
+- **Model-aware routing**: Instead of simply routing based on the path of the request, Gateway API Inference Extension allows you to route to models based on the model names. This is enabled by support for GenAI Inference API specifications (such as OpenAI API) in the gateway implementations such as in Envoy Proxy. This model-aware routing also extends to Low-Rank Adaptation (LoRA) fine-tuned models.
+
+- **Serving priority**: Gateway API Inference Extension allows you to specify the serving priority of your models. For example, you can specify that your models for online inference of chat tasks (which is more latency sensitive) have a higher priority than a model for latency tolerant tasks such as a summarization. 
+
+- **Model rollouts**:  Gateway API Inference Extension allows you to incrementally roll out new model versions by traffic splitting definitions based on the model names. 
+
+- **Extensibility for Inference Services**: Gateway API Inference Extension defines extensibility pattern for additional Inference services such as AI Safety checks or Semantic Caching. Gateway API Inference Extension.
```

Comment:
This is not supported by this project specifically, right? @robscott do you think we should mention this here?